lib/generate_layout.js

```diff
@@ -11,14 +9,44 @@ module.exports = pages => {
       <meta name="description" content="${page.description}">
       <meta name="viewport" content="width=device-width,initial-scale=1">
 	  <link href="/style.css" rel="stylesheet">
```

Comment:
that's a bug now 😃

Reply:
Jupp. I guess I have to read it from the manifest anyway 🤔

Reply:
Oh dear, what have I done...